### ISSUE TITLE
Gem Dragonborn not on list of races.

### ISSUE THREAD

**Issue by nboehret** [safety-orange/Imports-for-MPMB-s-Character-Sheet#77]
After importing the all_Wotc_published file and confirming that Fizban's races are in the file under the funciton `FToD_dragonborns_add` I tried to set the race to Gem Dragonborn but it does not show up on the drop down list or when typed. When typed it will automatically select the PHB Dragonborn. I am not seeing any Gem Dragonborn listed under Sources > Player Races but there is Dragonborn, Metallic listed with the source as ftod.

**Comment by Smashman** [comment]
Ensure the Fizban's source is enabled.

Then try inputting "Dragonborn, Gem". I just tried this on my sheet and it works correctly.

**Comment by safety-orange** [comment]
I can't reproduce this, it seems to be user error, so I'm closing this issue. You are welcome to reopen it if you can provide details on how to reproduce this issue with the latest version of the sheet (v13.1.0).

The most likely reason for an error (that I can think of) is that you have multiple import scripts, some of which are conflicting.

1. Please make sure you are using the latest version of the sheet (v13.1.0), you can [get it here for free](https://www.flapkan.com/#donation-select).
2. Import the latest version of [all_WotC_published.js](https://raw.githubusercontent.com/safety-orange/Imports-for-MPMB-s-Character-Sheet/master/WotC%20material/all_WotC_published.js), and only that file.
3. Don't change any of the included/excluded sources.
4. Then, try again to make a Gem Dragonborn by selecting "Dragonborn, Gem" from the race dropdown.
5. If the issue still persists, that you get  feel free to reopen this issue.

For instructions how to import a script, check out [this reddit post](https://www.reddit.com/r/mpmb/comments/jka96i/guide_to_get_the_sheets_and_add_content/).

**Comment by nboehret** [comment]
Just to fully close the loop on this: When I encountered this issue I had just installed the new version and the used the all_Wotc_published.js freshly downloaded and it was the only file. I had not changed any sources but "Dragonborn, Gem" was not showing up.

I followed the steps you laid out in case the issue was on me and it did work. My guess is I didn't hit the final 'Apply' but 'Cancel' instead, perhaps? I feel like it wouldn't show all the other races without something set as a source but I previously did have 13.0.8 and 13.0.4 installed and I am not sure it saves information somewhere that would be shared between versions, like locations of older all_WotC_published.js or something. Just speculating in case this matters to someone later.

Either way, thank you for helping me fix this!
